### PR TITLE
Log spam

### PR DESCRIPTION
**Pull request by lfarrell**

### CODE REVIEW

File: fcrepo-clients/src/main/java/edu/unc/lib/dl/fedora/ManagementClient.java [hunk 1]
```diff
@@ -339,6 +340,28 @@ public Datastream getDatastream(PID pid, String dsID, String asOfDateTime) throw
...
+
+		try (CloseableHttpResponse httpResp = httpClient.execute(method)) {
+			int statusCode = httpResp.getStatusLine().getStatusCode();
+
+			if (statusCode == 200) {
+				return true;
+			}
+
+			return false;
+		} catch (Exception e) {
```
Comment (harringj) [inline fcrepo-clients/src/main/java/edu/unc/lib/dl/fedora/ManagementClient.java]: Do we want to log any exceptions that might get thrown when execute() fails?

File: fcrepo-clients/src/main/java/edu/unc/lib/dl/fedora/ManagementClient.java [hunk 2]
```diff
@@ -339,6 +340,28 @@ public Datastream getDatastream(PID pid, String dsID, String asOfDateTime) throw
...
+		builder.pathSegment(pid.getPid());
+		builder.pathSegment("datastreams");
+		builder.pathSegment(dsID);
+
+		HttpHead method = new HttpHead(builder.build().encode().toUriString());
+
+		try (CloseableHttpResponse httpResp = httpClient.execute(method)) {
+			int statusCode = httpResp.getStatusLine().getStatusCode();
+
+			if (statusCode == 200) {
```
Comment (harringj) [inline fcrepo-clients/src/main/java/edu/unc/lib/dl/fedora/ManagementClient.java]: If we just want to know whether a ds exists, could there be cases where another 2xx code is returned that we would want to return true, or is 200 really the only signal of success here?

Comment (lfarrell) [inline fcrepo-clients/src/main/java/edu/unc/lib/dl/fedora/ManagementClient.java]: Seemed like 200 is the only one return, but could check for any 2xx code

File: fcrepo-clients/src/main/java/edu/unc/lib/dl/fedora/ManagementClient.java (hunk 1 above)
Comment (lfarrell) [inline fcrepo-clients/src/main/java/edu/unc/lib/dl/fedora/ManagementClient.java]: I'll leave that to Ben.

Comment (bbpennel) [inline fcrepo-clients/src/main/java/edu/unc/lib/dl/fedora/ManagementClient.java]: Yes that sounds like a good idea

Comment (lfarrell) [inline fcrepo-clients/src/main/java/edu/unc/lib/dl/fedora/ManagementClient.java]: Added logging on error. The most common we got was 500 for a malformed request to fedora.

File: fcrepo-clients/src/main/java/edu/unc/lib/dl/fedora/ManagementClient.java (hunk 2 above)
Comment (lfarrell) [inline fcrepo-clients/src/main/java/edu/unc/lib/dl/fedora/ManagementClient.java]: Looking at HTTP response codes it doesn't seem like the others would be relevant.

Comment (harringj) [inline fcrepo-clients/src/main/java/edu/unc/lib/dl/fedora/ManagementClient.java]: Cool, thanks for checking on that!

File: fcrepo-clients/src/main/java/edu/unc/lib/dl/fedora/ManagementClient.java (hunk 1 above)
Comment (harringj) [inline fcrepo-clients/src/main/java/edu/unc/lib/dl/fedora/ManagementClient.java]: 👍

**Pull request merged**